src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Program.cs

```diff
@@ -198,6 +198,12 @@ static CliRootCommand CreateCommandLineParser()
         Required = false
     };
 
+    var devKitRazorOutputPathOption = new CliOption<string?>("--devKitRazorOutputPath")
+    {
+        Description = "Full path to the Razor output path used with DevKit (optional).",
```

Comment:
This setting is not just for DevKit, right? I would expect it to apply to the C# extension whether DevKit was enabled or not. Perahsp `--razorSourceGeneratorPath` or `--razorServerPath` would be a more appropriate name?